### ISSUE TITLE
Can we restore only from local backup and not downloading it again from the source Firestore account?

### ISSUE THREAD

**Issue by hhimanshu** [willhlaw/node-firestore-backup-restore#5]
Currently, the backup and restore works together like   
```
➜  fs firestore-backup-restore -a source_service_account_key.json -B bakup --restoreAccountCredentials target_service_account_key.json
```  
This makes a dependency of `restore` on source account. This is not useful if we only want to restore based on the current backup on the filesystem. This makes the local storage not so useful.  

**Requirement**  
We should be able to restore based on `-B` and `-a2` flags only. Currently, it is not possible  

```
firestore-backup-restore -B bakup --restoreAccountCredentials target_service_account_key.json
Missing: accountCredentials - Google Cloud account credentials JSON file

  Usage: firestore-backup-restore [options]


  Options:

    -V, --version                            output the version number
    -a, --accountCredentials <path>          Google Cloud account credentials JSON file
    -B, --backupPath <path>                  Path to store backup.
    -a2, --restoreAccountCredentials <path>  Google Cloud account credentials JSON file for restoring documents.
    -P, --prettyPrint                        JSON backups done with pretty-printing.
    -h, --help                               output usage information
```

**Comment by willhlaw** [comment]
Good idea, @hhimanshu. I and others agree. I'm going to close this in favor of #3. Please take a look at that one and add any additional thoughts there please.

**Comment by willhlaw** [comment]
Restore from disk to firestore, -B and -a2, flags is available now as of `firestore-backup-restore@1.2.1`.
